### DISCUSSION TITLE
Can I install midnight commander?

### DISCUSSION THREAD

**Discussion by krama** [unifi-utilities/unifi-common#323] [Q&A]
The question is simple, is it possible to install and use, yes I understand that to me there will be questions "why for?" but still I wonder if it is possible or has anyone tried to do?

I mean the installation is not in the "unifi-os shell", but specifically in a shell like the other utilities

**Comment by KarlBaumann** [comment]
that was the first thing I installed on my UDM PRO SE.
`apt install mc`
